### PR TITLE
VBM-117: add build & version number on home screen

### PR DESCRIPTION
**Pull request by natata7**
Знімок екрана 2022-08-30 о 12 53 10

### CODE REVIEW

File: packages/mobile/src/screens/home/home.tsx [hunk 1]
```diff
@@ -1,10 +1,27 @@
 import React, { FC } from 'react';
-import { Text, ScreenWrapper } from '~/components/components';
+import { Text, ScreenWrapper, View } from '~/components/components';
+import DeviceInfo from 'react-native-device-info';
+import { globalStyles } from '~/styles/styles';
 
 const Home: FC = () => {
   return (
-    <ScreenWrapper>
+    <ScreenWrapper style={[globalStyles.justifyContentSpaceBetween]}>
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/home/home.tsx:8]:
```suggestion
    <ScreenWrapper style={globalStyles.justifyContentSpaceBetween}>
```

File: packages/mobile/src/screens/home/home.tsx [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+          globalStyles.py3,
+        ]}
+      >
+        <Text style={[globalStyles.fs12, globalStyles.mr1]}>BuildNumber:</Text>
+        <Text style={[globalStyles.fs12, globalStyles.mr5]}>
+          {DeviceInfo.getBuildNumber()}
+        </Text>
+        <Text style={[globalStyles.fs12, globalStyles.mr1]}>Version:</Text>
+        <Text style={globalStyles.fs12}>{DeviceInfo.getVersion()}</Text>
+      </View>
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/home/home.tsx:24]: Please extract it to a separate component

File: packages/mobile/src/components/build-info/build-info.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
+import { Text, View } from '~/components/components';
+import DeviceInfo from 'react-native-device-info';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/components/build-info/build-info.tsx:3]: Move it up, pls

**Review by AleksandrTermenzhy — APPROVED** [review]

File: packages/mobile/src/screens/home/home.tsx (hunk 1 above)
Comment (natata7) [reply packages/mobile/src/screens/home/home.tsx]: updated

File: packages/mobile/src/screens/home/home.tsx (hunk 2 above)
Comment (natata7) [reply packages/mobile/src/screens/home/home.tsx]: updated

File: packages/mobile/src/components/build-info/build-info.tsx (hunk 3 above)
Comment (natata7) [reply packages/mobile/src/components/build-info/build-info.tsx:2]: updated

**Pull request merged**